util/cron/nightlysubs.pm

```diff
@@ -117,7 +117,11 @@ sub endMailHeader {
 }
 
 sub endMailChplenv {
-    my $chplenv = `$chplhomedir/util/printchplenv --debug`;
+    my $ch = $chplhomedir;
```

Comment:
Is this the only time chplhomedir is used?  It's the only time I see it used in this file.  Would it be worthwhile doing this check when it was defined instead of later?